scaffolding-chef/README.md

```diff
@@ -0,0 +1,29 @@
+# scaffolding-base
+
+## Maintainers
+
+* The Habitat Maintainers: <humans@habitat.sh>
+
+## Usage
+
+A scaffolding for building Chef Policyfiles into executable habitat packages.
+
+```bash
+scaffold_policy_name="base"
+pkg_name="chef-base"
+pkg_origin=chefdorf
+pkg_version="0.1.0"
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=("Apache-2.0")
+pkg_description="The Chef $scaffold_policy_name Policy"
+pkg_upstream_url="http://chef.io"
+pkg_scaffolding="core/scaffolding-chef"
+```
+
+Will take a `policyfiles/base.rb` and build it. The scaffolding expects that
```

Comment:
The `repo` generator in ChefDK creates `policies` rather than `policyfiles`; maybe let's support both or standardise? I prefer `policies`, but that opinion is not strongly held.

Reply:
I prefer policyfiles here - it feels more expected given that we call the feature policyfiles.

Reply:
Let's change the generator then? like i said, i mostly don't care which, but they should be consistent.